### ISSUE TITLE
API/JSON Feed

### ISSUE THREAD

**Issue by jhammink** [KwnyPwny/cvecrowd.com#3]
It would be helpful to have some sort of API or JSON feed so people could poll the site for updates and filter out interesting new CVE's (and set up alerts or something). I'm using the NVD API for this now, but the buzz on Mastodon seems a much better indicator of the importance of the CVE than just the CVSS score.

**Comment by KwnyPwny** [comment]
Thanks for reaching out. I understand the usefulness of an API. I have some improvements to make to the UI and crawler, which I currently prioritize over API development. Once I'm there, I'll take a look.

**Comment by captn3m0** [comment]
An RSS feed would be great as well.

**Comment by KwnyPwny** [comment]
> An RSS feed would be great as well.

How should that RSS feed look like in your opinion? A daily update of CVEs being discussed?

**Comment by captn3m0** [comment]
Yeah, just a few feeds with links and text of the CVEs being discussed. Variants should allow for time-based slots, to let consumers decide:

1. Hourly Digest RSS Feed: A digest feed, with top-X (based on stars/Reblogs) posts, with at most one message per hour.
2. Daily RSS Feed: Same as above, but daily.
3. Firehose: Feed for all the posts, as they show up.

Different orgs might want to subscribe to them via different channels.

**Comment by KwnyPwny** [comment]
An API is now implemented on https://api.cvecrowd.com.

RSS Feed not yet.
